Web/HTML/computer-code/intro-var.md

```diff
@@ -11,29 +11,65 @@ category: must-know
 
 stub: true
 
-
 tags:
   - deep
-
-
+---
+# Intro var
 ---
 ## Content
 
-New content to go here. The author must be updated to match a valid Enki account.
+
+In HTML the `<var>` element is used to represent a name of a variable in either a programming context or a mathematical expression.
+
+**Note:**
+  - **var stands for The Variable element.**
+  - **Both the starting and ending tags are necessary.**
+
+Example:
+```
+<p>
+ <var>X</var> >
+ <var>Y</var>
+</p>
+```
+
+Result:
+<p>
+ <var>X</var> &gt;
+ Y
+</p>
+ 
+In the example above, the `<var>` element is used to create an inline element and to denote variable names in a mathematical expression.
+
+The default text style for the `<var>` element for most browsers is monospace and <i>italic</i>.
+
+However, you can edit this by using CSS like so:
+
+```
+var {
+  font: bold 20px "Times New Roman",
+  Times, serif;
+}
+```
 
 ---
 ## Practice
 
 When using the `<var>` element, which statements are true?
 
```

Comment:
If the user selects the correct elements but in the wrong order, they will still get it wrong. 
We don't have a proper control for "select all elements that are true". Normally when there are multiple true statements, what you want is to pick the one that's false:
"When using the <var> element, which statement is false?"

- false statement
- true statement
- true statement
- true statement